.github/workflows/checks.yml

```diff
@@ -75,7 +75,7 @@ jobs:
         env:
           GH_TOKEN: ${{ github.token }}
       - name: Install dfx
-        run: ./bin/dfx-software-dfx-install --version "$(jq -r .dfx dfx.json)"
+        uses: dfinity/setup-dfx@main
```

Comment:
Just checking - this does get the version from dfx.json, yes?

Reply:
Yes, it is: https://github.com/dfinity/setup-dfx

Reply:
Yes, since yesterday, on my request.